app/src/main/java/com/example/zava/mymobileapp/DB/DBHelp.java

```diff
@@ -0,0 +1,55 @@
+package com.example.zava.mymobileapp.DB;
+
+import android.content.ContentValues;
+import android.content.Context;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteOpenHelper;
+
+/**
+ * Created by Zava on 17.10.2018.
+ */
+
+public class DBHelp extends SQLiteOpenHelper {
+
+  public  DBHelp(Context c){
+    super(c, "Favourite", null, 1);
+  }
+
+  @Override
+  public void onCreate(SQLiteDatabase db) {
+    db.execSQL("create table album( id INTEGER PRIMARY KEY AUTOINCREMENT, albumName text, " +
+        "artistName text, albumImageURL text, albumReleaseDate text)");
+  }
+
+  @Override
+  public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
+//    db.execSQL("drop table if exists album");
+  }
+
+  public boolean insert(String albumName, String artistName, String
```

Comment:
put all params to one line or split per each line, now looks like u can put them in one line